src/rating/rating.spec.ts

```diff
@@ -582,6 +595,42 @@ describe('ngb-rating', () => {
       expect(element.nativeElement).toHaveCssClass('ng-untouched');
     });
 
+    it('should not update template driven form by clicking disabled control', () => {
+      const html = `
+          <ngb-rating [(ngModel)]="model" class="control" max="5"></ngb-rating>
+          <ngb-rating [(ngModel)]="model" class="control-disabled" max="5"></ngb-rating>`;
+
+      const fixture = createTestComponent(html);
+      const element = fixture.debugElement.query(By.css('.control'));
+      const disabledElement = fixture.debugElement.query(By.css('.control-disabled'));
+      let disabledRatingComp = <NgbRating>disabledElement.componentInstance;
+      disabledRatingComp.disabled = true;
```

Comment:
What is the point of doing this? we should just add the `disabled`:

```html
<ngb-rating [(ngModel)]="model" class="control-disabled" max="5" disabled></ngb-rating>
```

`ngModel` has a `disabled` input → https://angular.io/api/forms/NgModel